src/main/steps/c100-rebuild/other-proceedings/documentUpload/template.njk

```diff
@@ -6,6 +6,12 @@
 {% from "../../../common/fileupload/macro.njk" import fileUpload %}
 
 {% extends "common/page.njk" %}
+
+{% block pageTitle %}
```

Comment:
Not required, when the title is constructed.

Reply:
Changed to set the title in content.ts